Assets/HexMap.cs

```diff
@@ -215,7 +188,9 @@ public Hex GetAcceptableStartPosition()
             int impassibleTiles = 0;
             foreach (Hex h in neighbors)
             {
-                if (h.GetTerrain() == Terrain.MOUNTAIN || h.GetTerrain() == Terrain.WATER)
+                if (h == null)
+                    Debug.LogWarning("Null neighbor, something is borked");
```

Comment:
Could you explain what you don't like about this?  Is it the debug warning, the checking for null, or what?  Would you prefer I wrapped this in a try/catch type statement?